### PR TITLE
include Addition (Add) blending mode + Officially drop support for Node 8 

### PR DESCRIPTION
**Pull request by GlitchyPSIX**
# What's Changing and Why
Addition mode is being... *added* to the list of compositing modes. Minor change (new feature), fulfills the old request from #724 because I also needed it)
## What else might be affected
Nothing else in the `core` package should be affected.
## Tasks

- [x] Add tests [Not necessary]
- [x] Update Documentation
- [x] Update `jimp.d.ts`
- [x] Add [SemVer](https://semver.org/) Label

<details>
  <summary>📦 Published PR as canary version: <code>0.14.0-canary.904.976.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @jimp/cli@0.14.0-canary.904.976.0
  npm install @jimp/core@0.14.0-canary.904.976.0
  npm install @jimp/custom@0.14.0-canary.904.976.0
  npm install jimp@0.14.0-canary.904.976.0
  npm install @jimp/plugin-blit@0.14.0-canary.904.976.0
  npm install @jimp/plugin-blur@0.14.0-canary.904.976.0
  npm install @jimp/plugin-circle@0.14.0-canary.904.976.0
  npm install @jimp/plugin-color@0.14.0-canary.904.976.0
  npm install @jimp/plugin-contain@0.14.0-canary.904.976.0
  npm install @jimp/plugin-cover@0.14.0-canary.904.976.0
  npm install @jimp/plugin-crop@0.14.0-canary.904.976.0
  npm install @jimp/plugin-displace@0.14.0-canary.904.976.0
  npm install @jimp/plugin-dither@0.14.0-canary.904.976.0
  npm install @jimp/plugin-fisheye@0.14.0-canary.904.976.0
  npm install @jimp/plugin-flip@0.14.0-canary.904.976.0
  npm install @jimp/plugin-gaussian@0.14.0-canary.904.976.0
  npm install @jimp/plugin-invert@0.14.0-canary.904.976.0
  npm install @jimp/plugin-mask@0.14.0-canary.904.976.0
  npm install @jimp/plugin-normalize@0.14.0-canary.904.976.0
  npm install @jimp/plugin-print@0.14.0-canary.904.976.0
  npm install @jimp/plugin-resize@0.14.0-canary.904.976.0
  npm install @jimp/plugin-rotate@0.14.0-canary.904.976.0
  npm install @jimp/plugin-scale@0.14.0-canary.904.976.0
  npm install @jimp/plugin-shadow@0.14.0-canary.904.976.0
  npm install @jimp/plugin-threshold@0.14.0-canary.904.976.0
  npm install @jimp/plugins@0.14.0-canary.904.976.0
  npm install @jimp/test-utils@0.14.0-canary.904.976.0
  npm install @jimp/bmp@0.14.0-canary.904.976.0
  npm install @jimp/gif@0.14.0-canary.904.976.0
  npm install @jimp/jpeg@0.14.0-canary.904.976.0
  npm install @jimp/png@0.14.0-canary.904.976.0
  npm install @jimp/tiff@0.14.0-canary.904.976.0
  npm install @jimp/types@0.14.0-canary.904.976.0
  npm install @jimp/utils@0.14.0-canary.904.976.0
  # or 
  yarn add @jimp/cli@0.14.0-canary.904.976.0
  yarn add @jimp/core@0.14.0-canary.904.976.0
  yarn add @jimp/custom@0.14.0-canary.904.976.0
  yarn add jimp@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-blit@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-blur@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-circle@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-color@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-contain@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-cover@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-crop@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-displace@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-dither@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-fisheye@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-flip@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-gaussian@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-invert@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-mask@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-normalize@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-print@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-resize@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-rotate@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-scale@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-shadow@0.14.0-canary.904.976.0
  yarn add @jimp/plugin-threshold@0.14.0-canary.904.976.0
  yarn add @jimp/plugins@0.14.0-canary.904.976.0
  yarn add @jimp/test-utils@0.14.0-canary.904.976.0
  yarn add @jimp/bmp@0.14.0-canary.904.976.0
  yarn add @jimp/gif@0.14.0-canary.904.976.0
  yarn add @jimp/jpeg@0.14.0-canary.904.976.0
  yarn add @jimp/png@0.14.0-canary.904.976.0
  yarn add @jimp/tiff@0.14.0-canary.904.976.0
  yarn add @jimp/types@0.14.0-canary.904.976.0
  yarn add @jimp/utils@0.14.0-canary.904.976.0
  ```
</details>

### CODE REVIEW

**Comment by GlitchyPSIX** [comment]
No clue why it's failing on node8. It says the GIF export reached a timeout but I don't think that's my fault as every other check passed...

Maybe this is what the commits meant by "browser version being broken"?

**Comment by hipstersmoothie** [comment]
You can drop the node 8 config. we can officially drop support with this PR

File: packages/core/package.json [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-  },
-  "devDependencies": {
-    "should": "^13.2.3"
-  },
-  "xo": false,
-  "publishConfig": {
-    "access": "public"
-  }
-}
+    "name": "@jimp/core",
```
Comment (hipstersmoothie) [inline packages/core/package.json:2]: revert changes to this file please

**Comment by GlitchyPSIX** [comment]
> Just fix the package.json and this is good to merge

This good?
(First time contributing to another library, sorry if I mess up every now and then)

**Comment by hipstersmoothie** [comment]
@GlitchyPSIX Perfect!

No worries, not all repos have their releases automated in the same way jimp does (using https://github.com/intuit/auto)

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in `v0.14.0` :rocket:

**Pull request merged**